PhysicsTools/PatAlgos/plugins/JetCorrFactorsProducer.cc

```diff
@@ -232,12 +232,9 @@ JetCorrFactorsProducer::produce(edm::Event& event, const edm::EventSetup& setup)
 					    << "the sequence.";
     }
     for(unsigned int idx=0; idx<corrLevel->second.size(); ++idx){
-      bool flavorDependent=false;
       std::vector<float> factors;
-      if(flavorDependent ||
-	 corrLevel->second[idx].find("L5Flavor")!=std::string::npos ||
+      if(corrLevel->second[idx].find("L5Flavor")!=std::string::npos ||
```

Comment:
@ahinzmann, @rappoccio: Could you please offer some guidance about flavor-dependent corrections? We are thinking of fixing PhysicsTools/PatAlgos/plugins/JetCorrFactorsProducer.cc so it will apply flavor-dependent corrections for L6UE, and we need to know if this change would be appropriate.